styles.css

```diff
@@ -0,0 +1,365 @@
+body {
+  font-family: Public Sans;
+  font-size: 18px;
+}
+
+header {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  font-weight: 600;
+  color: #919095;
+}
+
+.header_navbar li {
+  display: inline;
+  padding: 20px;
+}
+
+.header_navbar li:hover {
+  transition: 100ms;
+  border-bottom: 5px solid transparent;
+  border-image: linear-gradient(to right, #31cf72, #2bb9d1);
+  border-image-slice: 1;
+}
+
+.request_button {
+  height: 45px;
+  width: 170px;
+  background-image: linear-gradient(to right, #31cf72, #2bb9d1);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 25px;
+  color: #ffffff;
+  font-weight: 700;
+}
+
+.request_button:hover {
+  opacity: 0.5;
+  transition: 100ms;
+}
+
+.hidden_desktop {
+  display: none;
+}
+
+h1 {
+  font-size: 45px;
+}
+
+.banner {
+  background-color: #fafafa;
+}
+
+.banner_image_desktop {
+  height: 600px;
+  width: 150%;
+  object-fit: cover;
+  object-position: 50% 113%;
+}
+
+#benefits {
+  background-color: #f4f5f7;
+}
+
+.benefits_boxes div p {
+  font-size: 0.8em;
+}
+
+.articles {
+  height: 600px;
+  background-color: #fafafa;
+}
+
+.articles h3:hover {
+  color: #34cf95;
+}
+
+.article_section_title {
+  margin: 50px;
+}
+
+.container {
+  height: 500px;
+}
+
+.container div {
+  height: 400px;
+  width: 250px;
+  border-radius: 10px;
+  background-color: #ffffff;
+}
+
+.container div p {
+  margin-left: 20px;
+  margin-right: 20px;
+  font-size: 0.7em;
+}
+
+.container div h3 {
```

Comment:
Tutaj dopowiem donośnie tego wyżej.
Jeśli mamy taką sytuację że mamy style dla p i h3 to warto dodać np do h3 klasę section-heading a do p section-description wtedy unikniemy takiego zagnieżdżania przez diva